docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -1,319 +1,402 @@
+- `"Try" Operator`_
 
 See :ref:`storm-ref-syntax` for an explanation of the syntax format used below.
 
 See :ref:`storm-ref-type-specific` for details on special syntax or handling for specific data types.
 
-Simple Lifts
-------------
-
-"Simple" lifts refers to the most "basic" lift operations. That is, operations to retrieve a set of nodes based on:
+.. _lift-form:
 
-- A specific primary, secondary, or universal property.
-- A specific primary, secondary, or universal property value.
-- A specific tag or tag property.
+Lift by Form
+------------
 
-The only difference between "simple" lifts and "lifts using comparison operators" is that we have defined simple lifts as those that use the equals ( ``=`` ) comparator, which is the easiest comparator to use to explain basic lift concepts.
+A "lift by form" operation returns all nodes for the specified :ref:`data-form`. This type of lift
+requires the name of the form whose nodes you want to lift.
 
 **Syntax:**
 
-*<form>* [ **=** *<valu>* ]
-
-*<form>* **=** *<valu>* **:** *<prop>* [ **=** *<pval>* ] 
-
-**#** *<tag>* [ **:** *<tagprop>* [ *<operator>* *<pval>* ] ] 
-
-**#:** *<tagprop>* [ *<operator>* *<pval>* ]
+*<form>*
 
 **Examples:**
 
-*Lift by primary property (<form>):*
+*Lift all FQDN (inet:fqdn) nodes:*
 
-- Lift all domain nodes:
+.. storm-pre:: [ inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com ]
+.. storm-pre:: inet:fqdn
+::
+  
+  inet:fqdn
 
+*Lift all nodes representing articles (media:news nodes):*
 
-.. storm-pre:: [inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:fqdn
+.. storm-pre:: [ (media:news=* :publisher:name=eset :published=2016/10/20 :title='dissection of sednit espionage group') (media:news=* :publisher:name='trend micro' :published=2023/09/18 :title='earth lusca employs new linux backdoor, uses cobalt strike for lateral movement') ]
+.. storm-pre:: media:news
+::
+  
+  media:news
 
+**Usage notes:**
 
-- Lift all mutex nodes:
+- A "lift by form" returns **all** nodes of that form in Synapse. In a production instance of Synapse, lifting
+  **all** nodes of a commonly used form (e.g., ``inet:fqdn`` or ``inet:ipv4``) may return thousands or tens of
+  thousands of nodes.
+- A "lift by form" can be used with the Storm :ref:`storm-limit` command to return only a specified number of
+  nodes (for example, to view a sample of available data).
 
 
-.. storm-pre:: [it:dev:mutex="006ae432" it:dev:mutex="hello"]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: it:dev:mutex
+.. _lift-prop:
 
+Lift by Property
+----------------
 
-*Lift a specific node (<form> = <valu>):*
+A "lift by property" operation returns all nodes that **have** the specified :ref:`data-props` set, regardless of the property value.
+This type of lift requires the full name (i.e., the combined form and property name) of the property you want to use to lift the nodes.
```

Comment:
Universal properties don't require a form when lifting, not sure if that is worth clarifying here or if the "Lift by Universal Property" section is sufficient.

Reply:
Clarified and added "optional" brackets [ ] around *<form>* for the universal syntax.